pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
> Are you sure it's only the GS plugin that still reports it's initialised? It could be that all the plugins have not been shutdown yet.

Derp, it seems you're right. All the plugins seem to be initialized even after a shutdown event.

 I'll make a blog post later explaining on how I screwed it up thinking only the GS plugin was initialized. Jokes aside, shouldn't we then fix the issue by properly shutting down the plugins when ``Shutdown`` is pressed ?